srcpkgs/libgxps/template

```diff
@@ -11,8 +11,14 @@ short_desc="GObject base library for XPS documents"
 maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
 license="LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Projects/libgxps"
-distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=412b1343bd31fee41f7204c47514d34c563ae34dafa4cc710897366bd6cd0fae
+distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz
+ https://github.com/ndowens/voidpatches/archive/libgxps.tar.gz"
```

Comment:
Just include that in this repo, it's not that big.

Reply:
> Just include that in this repo, it's not that big.

Ok. Was thinking it might be a little too big